docs/source/conf.py

```diff
@@ -283,3 +285,53 @@
 
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 #texinfo_show_urls = 'footnote'
+
+class HTML5Translator(SphinxHTML5Translator):
+    """
+    This is our custom HTML5Translator which extends the one provided by Sphinx.
+    """
+    STRONG_TO_H3_HEADINGS = [
+        'Example',
+        'Examples',
+        'Exceptions',
+        'Request Syntax',
+        'Response Structure',
+        'Response Syntax',
+        'Structure',
+        'Syntax'
+    ]
+
+    def visit_admonition(self, node, name=''):
+        """Uses the h3 tag for admonition titles instead of the p tag"""
+        self.body.append(self.starttag(
+            node, 'div', CLASS=('admonition ' + name)))
+        if name:
+            title = (
+                f"<h3 class='admonition-title'>"
+                f"{admonitionlabels[name]}</h3>"
+            )
+            self.body.append(title)
+
+    def visit_strong(self, node):
+        """
+        Opens the h3 tag for a specific set of words/phrases and opens the
+        strong tag for all others.
+        """
```

Comment:
Another doc string convention in Python is to have a brief sentence on one line, then a line break, then some more details if needed.

I'm not sure if this first sentence is exactly accurate, but something along these lines:

```suggestion
        """Visit a strong HTML element.
        
        Opens the h3 tag for a specific set of words/phrases and opens the
        strong tag for all others.
        """
```